.github/workflows/CI.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  Tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node-version: [8, 10, 12, 13]
```

Comment:
I think we should exclude Node version `8`. It is pretty legacy.

Reply:
@sQVe That's not a bad idea. I think we should use your first idea of staying with the matrix but only using one node version since we already had a reported issue on windows.